src/service/ConfigLoader.ts

```diff
@@ -383,13 +383,10 @@ export class ConfigLoader {
         const customPresetObject = params.customPresetObject;
         const oldPresetData = params.oldPresetData;
         const customPresetFileObject = this.loadCustomPreset(customPreset, params.password);
-        const preset =
-            params.preset ||
-            params.customPresetObject?.preset ||
-            customPresetFileObject?.preset ||
-            oldPresetData?.preset ||
-            Preset.bootstrap;
-
+        const preset = params.preset || params.customPresetObject?.preset || customPresetFileObject?.preset || oldPresetData?.preset;
```

Comment:
Whats the order of precedence for the presets?  Should ``customPresetFileObject`` comes before ``customPresetObject``?

Reply:
if the user is providing both, a file and a custom preset object, the custom preset object is priority. This is the case when you use bootstrap as a library. 

I would probably need to revisit this, especially in private networks when the network.yml is provided instead of built-in bootstrap.

Reply:
if you don't already, consider adding a build section to the readme (if you don't already) about how to generate these files and the expected test reporting files too. can be separate PR.